test/smoke-fort/milestone-3-babel-pdot/OpenMPTargetStream.F90

```diff
@@ -0,0 +1,134 @@
+              stop
+            else
+              call omp_set_default_device(dev)
+            end if
+        end subroutine set_device
+
+        subroutine alloc(array_size)
+            implicit none
+            integer(kind=StreamIntKind) :: array_size
+            integer :: err
+            N = array_size
+            allocate( A(1:N), B(1:N), C(1:N), stat=err)
+#ifndef USE_MANAGED
+            !$omp target enter data map(alloc: A,B,C)
+#endif
+            if (err .ne. 0) then
+              write(*,'(a20,i3)') 'allocate returned ',err
+              stop 1
+            endif
+        end subroutine alloc
+
+        subroutine dealloc()
+            implicit none
+            integer :: err
+#ifndef USE_MANAGED
+            !$omp target exit data map(delete: A,B,C)
+#endif
+            deallocate( A, B, C, stat=err)
+            if (err .ne. 0) then
+              write(*,'(a20,i3)') 'deallocate returned ',err
+              stop 1
+            endif
+        end subroutine dealloc
+
+        subroutine init_arrays(initA, initB, initC)
+            implicit none
+            real(kind=REAL64), intent(in) :: initA, initB, initC
+            integer(kind=StreamIntKind) :: i
+ !           !$omp target teams distribute parallel do simd
+            do i=1,N
+               A(i) = initA
+               B(i) = initB
+               C(i) = initC
+            end do
+        end subroutine init_arrays
+
+        subroutine read_arrays(h_A, h_B, h_C)
+            implicit none
+            real(kind=REAL64), intent(inout) :: h_A(:), h_B(:), h_C(:)
+            integer(kind=StreamIntKind) :: i
+            ! this might need to use a copy API instead...
+ !           !$omp target teams distribute parallel do simd
+            do i=1,N
+               h_A(i) = A(i)
+               h_B(i) = B(i)
+               h_C(i) = C(i)
+            end do
+        end subroutine read_arrays
+
+        subroutine copy()
+        end subroutine copy
+
+        subroutine add()
+        end subroutine add
+
+        subroutine mul(startScalar)
+            real(kind=REAL64) :: startscalar
+        end subroutine mul
+
+        subroutine triad(startScalar)
+            real(kind=REAL64) :: startscalar
+        end subroutine triad
+
+        subroutine nstream(startScalar)
+            implicit none
+            real(kind=REAL64), intent(in) :: startScalar
+            real(kind=REAL64) :: scalar
+            integer(kind=StreamIntKind) :: i
+            scalar = startScalar
+        !   !$omp target teams distribute parallel do simd
+            do i=1,N
+               A(i) = A(i) + B(i) + scalar * C(i)
+            end do
+        end subroutine nstream
+
+        function dot() result(s)
+            implicit none
+            real(kind=REAL64) :: s
+            integer(kind=StreamIntKind) :: i
+            s = real(0,kind=REAL64)
+            !$omp target teams distribute parallel do simd reduction(+:s) map(tofrom:s)
```

Comment:
Looks like you started with milestone-3-babel-drop1 which had a couple workarounds that are no longer needed.
- Remove the add map(tofrom:s)